apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,408 @@
         "    print()\n",
         "    print(X.used_shape())"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "## Ingesting multiple datasets to a SOMA Experiment"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "Finally, we'll demonstrate combining multiple new AnnDatas into one experiment.\n",
+        "\n",
+        "The flow is pretty similar to the above:\n",
+        "\n",
+        "1. One call to `register_anndatas` or `register_h5ads` (passing all new AnnDatas/h5ads)\n",
+        "2. One call to `from_anndata`/`from_h5ad` *for each new AnnData*"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 15,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "def make_new_ad(when, scale, obs_id_suffix):\n",
+        "    ad = ad1.copy()\n",
+        "    ad.obs.index = [e.replace(\"-1\", obs_id_suffix) for e in ad.obs.index]\n",
+        "    ad.obs[\"when\"] = [when] * len(ad.obs)\n",
+        "    ad.X *= scale\n",
+        "    return ad\n",
+        "\n",
+        "new_ads = [\n",
+        "    make_new_ad(when, scale, f\"-{idx + 3}\")\n",
+        "    for idx, (when, scale)\n",
+        "    in enumerate({\n",
+        "        \"Wednesday\": 20,\n",
+        "        \"Thursday\": 30,\n",
+        "        \"Friday\": 40,\n",
+        "    }.items())\n",
+        "]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "Ingesting the new AnnDatas, as before (but this time to a fresh/empty `/tmp` location):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 16,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "'/tmp/append-example-20240517-204944'"
+            ]
+          },
+          "execution_count": 16,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "stamp = datetime.datetime.today().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "experiment_uri = f\"/tmp/append-example-{stamp}\"\n",
+        "experiment_uri"
+      ]
+    },
+    {
+      "cell_type": "code",
```

Comment:
Add:

Here we'll register all the AnnData objects. Note that the SOMA Experiment doesn't exist yet, so we pass `experiment_uri=None` to signify that.

Reply:
done